test/decl/init/cf-types.swift

```diff
@@ -14,10 +14,27 @@ extension CGMutablePath {
   public convenience init(toss: Bool) throws { // expected-error{{convenience initializers are not supported in extensions of CF types}}
     self.init()
   }
+
+  public init(simple: Bool) { // expected-error{{designated initializer cannot be declared in an extension of 'CGMutablePath'}}
+                              // expected-error @-1 {{designated initializer for 'CGMutablePath' cannot delegate (with 'self.init')}}
```

Comment:
You should also check that we don't actually emit any fix-its with this diagnostics. You can do this by adding `{{none}}` at the end of this check. So,

```suggestion
                              // expected-error @-1 {{designated initializer for 'CGMutablePath' cannot delegate (with 'self.init')}}{{none}}
```

You'll have to do it for both the new errors that you've added.

Reply:
Done